src/neural/network_cudnn.cu

```diff
@@ -997,6 +997,10 @@ class CudnnNetwork : public Network {
       }
     }
 
+    printf(
```

Comment:
Should go to stderr if it's even a good idea. But probably it's fine without it.
So let's drop it! Sorry for making you make this PR, it's probably good to go without the message.